### ISSUE TITLE
Complete Hyperparameters

### ISSUE THREAD

**Issue by GeorgeCazenavette** [GeorgeCazenavette/mtt-distillation#7]
Hi all,

Here is a _complete_ set of all the hyper parameters used to generate the results we present in the paper. 
This will be added to the paper when we upload v2 to arXiv.

Note that 
"Learning Rate (Pixels)" corresponds to `--lr_img`, 
"Learning Rate (Step Size)" corresponds to `--lr_lr`,
and 
"Starting Synthetic Step Size" corresponds to `--lr_teacher`.

Also, these hyperparameters for each dataset and support size were not found via an exhaustive search, so let us know if you happen to find a more optimal set!

Please let me know if there's anything else I can do to help or improve the repo :)

Thanks,

George

![image](https://user-images.githubusercontent.com/18726777/161732602-88798d2f-4577-4e48-8ee4-0dae8350bea6.png)
